src/Main.cpp

```diff
@@ -1,26 +1,37 @@
 #include "Scene.h"
 #include "MusicPlayer.h"
+#include <iostream>
```

Comment:
I think you forgot to delete it after some temp logging :)

Reply:
oh damn, forgot about it again, will delete next time, i don't wanna commit just to delete one line